native/src/modules/endpoint/sagas/__tests__/fetchResourceCache.spec.js

```diff
@@ -61,19 +62,12 @@ describe('fetchResourceCache', () => {
     const categoriesBuilder = new CategoriesMapModelBuilder(city, language)
     const fetchMap = categoriesBuilder.buildFetchMap()
 
+    FetcherModule.currentlyFetching = true
     return expectSaga(fetchResourceCache, city, language, fetchMap, dataContainer)
-      .provide({
-        call: (effect, next) => {
-          if (effect.fn.name === 'fetchAsync') {
-            throw new Error('Jemand hat keine 4 Issues geschafft!')
-          }
-          return next()
-        }
-      })
       .put({
         type: 'FETCH_RESOURCES_FAILED',
         params: {
-          message: 'Error in fetchResourceCache: Jemand hat keine 4 Issues geschafft!',
```

Comment:
I think this test was different before. The test should provide what happens when `fetchAsync` is called. Then it asserts that the `FETCH_RESOURCES_FAILED` is returned.

Now it returns for some reason Already fetching.

Reply:
It can not reach "fetchAsync" any more, because before creating the channel I added a new "throw error"   

````
if (FetcherModule.currentlyFetching) {
  throw new Error('Already fetching!')
}
const progressTask = yield fork(watchOnProgress)
const results = yield call(new FetcherModule().fetchAsync, targetFilePaths)
````